pagopa/io-react-native-proximity
# [SIW-756] Add `io-react-native-crypto` with hex signature

### Short description

This PR adds a custom `io-react-native-crypto` version which use `unpackBerEncoderASN1` util and a custom signing method. Originally `io-react-native-crypto` use a message payload to sign as UTF8 data charset encoded. In COSE signing flow the message is encoded in hex string. 

#### List of Changes
 - Update `io-react-native-crypto`: this is a temp solution a PR will be opened to update that package
 - Update cose-js to use TEE to sign
 - Update example app

#### How Has This Been Tested?

- Start the example app
- Start BLE
- Try `Sign with COSE`

On the console should appear message related to signed message (you could try to use a different payload) and the verify.

#### Screenshots (if appropriate):
